docs/Developer-Guide_Build-Switches.md

```
@@ -186,6 +186,19 @@ When choosing `ROOTFS_TYPE=btrfs`, select `btrfs` filesystem compression method
 
     The script does not check the legality of the input variable (compression ratio). Input like `zlib:1234` is legal to the script but illegal to the kernel. Beware that setting this option does affect image creation only (shrinking disk size) and will not adjust `/etc/fstab`, so it is up to the user to later edit `/etc/fstab` if compression in daily operation is also wanted (beware of severe performance penalties with random IO patterns and heavy compression algorithms!).
 
+**BTRFS_ROOT_SUBVOLUME**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)